frontend/packages/console-dynamic-plugin-sdk/src/api/core-api.ts

```diff
@@ -101,3 +113,13 @@ export const k8sDelete: K8sDelete = require('@console/dynamic-plugin-sdk/src/uti
   .k8sDeleteResource;
 export const k8sList: K8sList = require('@console/dynamic-plugin-sdk/src/utils/k8s')
   .k8sListResource;
+export const getReference: GetReference = require('@console/dynamic-plugin-sdk/src/utils/k8s')
+  .getReference;
+export const getReferenceForModel: GetReferenceForModel = require('@console/dynamic-plugin-sdk/src/utils/k8s')
+  .getReferenceForModel;
+export const getAPIVersionForModel: GetAPIVersionForModel = require('@console/dynamic-plugin-sdk/src/utils/k8s')
+  .getAPIVersionForModel;
+export const getGroupVersionKindForResource: GetGroupVersionKindForResource = require('@console/dynamic-plugin-sdk/src/utils/k8s')
+  .getGroupVersionKindForResource;
+export const getGroupVersionKindForReference: GetGroupVersionKindForReference = require('@console/dynamic-plugin-sdk/src/utils/k8s')
+  .getGroupVersionKindForReference;
```

Comment:
I think there is no reason to use `require` since the functions are already in `dynamic-plugin-sdk` . Can't you just import them directly ?